plugins/discovery-gce/src/test/resources/org/elasticsearch/discovery/gce/compute/v1/projects/nodeswithpagination/zones/europe-west1-b/instances?pageToken=next-token

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
I believe question marks are not allowed in filenames on Windows, hence the Windows tests fail. 

What do you think about replacing the question mark with `%3F` from https://en.wikipedia.org/wiki/Percent-encoding and then make sure the question mark gets encoded in `GceMockUtils#readJsonResponse`?

```java
String mockFileName = Strings.replace(url, urlRoot, "")
            .replace("?", "%3F")
```

Reply:
thanks @arteam  ! fixed !